lib/manager/components/deployment/DeploymentConfigurationsPanel.js

```diff
@@ -31,6 +31,16 @@ const TRIP_PLANNER_VERSIONS = [
   { label: 'OTP 2.X', value: 'OTP_2' }
 ]
 
+function calculateCustomFileKey (customFile: CustomFile, idx: number): string {
+  return `${
+    customFile.filename || ''
+  }-${
+    customFile.contents || ''
```

Comment:
Would a timestamp or a uuid better than the file contents for identification purposes? (If it's a uuid, we won't need the other attributes to generate a unique key)

Reply:
This is the tricky thing about the custom files since they're saved as an array within the deployment object. Perhaps they should be in their own table and then they would have IDs. But since they don't, I think it makes sense to use these items to create a composite key.